extractor/main.go

```diff
@@ -250,8 +250,8 @@ func extractAddMessages(filePath string) ([]vm.MsgAddPackage, error) {
 			line = tempBuf
 		}
 
-		if err := amino.UnmarshalJSON(line, &tx); err != nil {
-			fmt.Errorf("Error while parsing amino JSON at line: %w\nLine:%s\n", err, line)
+		if err := amino.UnmarshalJSON(line, &txData); err != nil {
+			_ = fmt.Errorf("Error while parsing amino JSON at line: %w\nLine:%s\n", err, line)
```

Comment:
the same, is that for the linter? you don't need to do that normally.

Reply:
Bbeside that, why this `Errorf` isn't assign to something ?

Reply:
In this case, we do not want to stop the execution if there is an error

Reply:
@gfanton is right, that error won't be printed on stdout.

Reply:
Hmmm, alright, let me see what I can do :)

Reply:
Implemented slog: [b4ebcf6](https://github.com/gnolang/tx-exports/pull/12/commits/b4ebcf677ce21f9ef4ee3c19443e5f98da358928)